_rules/image-filename-as-accessible-name-9eb3f6.md

```diff
@@ -53,7 +53,7 @@ The `img` element's [accessible name][] uses the filename which accurately descr
 
 #### Passed Example 2
 
-The `img` element's [accessible name][] includes the filename, which in combination with the text content of the `a` element accurately describes the image.
+The `img` element's [accessible name][] includes the filename, which in combination with the text content of the `<a>` element accurately describes the image.
```

Comment:
Nope. We don't put element name in brackets.
I assume the an/a checker is thrown out by the "a element" look-alike…